src/dbt/kipptaf/models/surveys/intermediate/int_surveys__survey_responses.sql

```diff
@@ -47,3 +52,84 @@ left join
     {{ ref("stg_reporting__terms") }} as rt
     on rt.name = fr.info_title
     and date(fr.last_submitted_time) between rt.start_date and rt.end_date
+
+union all
+
+select
+    safe_cast(sr.survey_id as string) as survey_id,
+    sr.survey_title as survey_title,
+    safe_cast(sr.response_id as string) as survey_response_id,
+    coalesce(
+        sr.response_string_value,
+        sr.response_map_value,
+        sr.response_option_value,
+        sr.response_rank_value
+    ) as answer,
```

Comment:
refactor to `base_alchemer__survey_results`

Reply:
hey, i'm not clear on what to change -- the coalesce? data types?

Reply:
the coalesce, it's reused a bunch